packages/stimulus-language-server/package.yaml

```
@@ -0,0 +1,22 @@
+---
+name: stimulus-language-server
+description: Intelligent Stimulus tooling
+homepage: https://github.com/marcoroth/stimulus-lsp
+licenses:
+  - MIT
+languages:
+  - Blade
+  - HTML
+  - PHP
+  - Ruby
+categories:
+  - LSP
+
+source:
+  id: pkg:npm/stimulus-language-server@0.2.1
+
+schemas:
+  lsp: vscode:https://raw.githubusercontent.com/marcoroth/stimulus-lsp/main/package.json
```

The configuration schema seems to only contain a single VSCode-only setting, but we might as well include it. To make sure the correct version of the schema is installed we can interpolate the `version` that is being installed:

```suggestion
  lsp: vscode:https://raw.githubusercontent.com/marcoroth/stimulus-lsp/v{{version}}/package.json
```